common/build/eslint-config-fluid/src/test/mockFiles/no-member-release-tags/mockInterface.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @internal
+ */
+interface MockInteface {
+	/**
+	 * @alpha
+	 */
+	invalidAlpha: string;
+
+	/**
+	 * @beta
+	 */
+	invalidBeta: string;
+
+	/**
+	 * @public
+	 */
+	invalidPublic: string;
+
+	/**
+	 * @internal
+	 */
+	invalidInternal: number;
+
+	// @internal
+	invalidInternalLine: number;
+
+	// This should be a valid implementation.
+	validComment: number;
+
+	valid: number;
+
+	validFunction(): boolean;
+
+	/**
+	 * @public
+	 */
+	invalidFunction(): boolean;
+}
+
+/**
+ * @public
+ */
+interface MockIntefaceTwo {
+	/**
+	 * @alpha
+	 */
+	invalidAlpha: string;
+
+	valid: number;
+}
```

Comment:
Are we getting something from this one that we're not getting from testing the one above? Feels to me like we could remove this one and would still have a complete test suite.

Reply:
This also has a somewhat _similar_ reasoning as the inclusion of release tag for the parent node. 

More often than not, multiple classes live in a single file, thus I wanted to verify that the linter is able to pick multiple parent nodes inside a single file.